### PR TITLE
apply random seed

### PR DESCRIPTION
**Pull request by plataKwon**
Hello, I try to fix some issues.

When I ran the model, I kept getting different results.
It is beacuase your model was not using random seeds.
So I modified the code to apply the random seed.
Please check.

Thanks.

### CODE REVIEW

**Comment by shenweichen** [comment]
Can you provide an example of runing a  model to get different results?

**Comment by plataKwon** [comment]
As the simplest example, you can run the example code(classification:criteo) from this site (https://deepctr-torch.readthedocs.io/en/latest/Examples.html)
I added random_state to the 41 line of code to avoid problems outside the model.
```
train, test = train_test_split(data, test_size=0.2, random_state=1024)
```

Nevertheless, every time I run the code, the output is different.
We should be able to get the same value if the parameters are the same, even if we run the train/test code multiple times.

Of course, I can get the same result by adding one line in the my executing code.
```
torch.manual_seed(seed)
```
But your code doesn't use seed parameter(even though the code is getting the seed variable as a parameter), so I think it is good to use the seed variable internally.
